src/pages/onboarding/components/steps/optionGroup/MultiOptionGroup.tsx

```
@@ -34,15 +36,16 @@
   isRequiredFurniture,
   currentActivityLabel,
   requiredFurnitureLabels = [],
+  useId = false,
 }: MultiOptionGroupProps<T>) => {
-  const handleClick = (optionCode: T) => {
+  const handleClick = (optionCode: T | number) => {
     // 필수 가구인 경우 클릭 무시
     if (isRequiredFurniture && isRequiredFurniture(optionCode)) {
       return;
     }
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)